modules/auxiliary/scanner/http/citrix_directory_traversal.rb

```diff
@@ -0,0 +1,79 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  # Exploit mixins should be called first
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::WmapScanServer
+  # Scanner mixin should be near last
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize
+    super(
+      'Name'        => 'Citrix ADC Directory Traversal',
+      'Description' => 'This module exploits a directory traversal vulnerability (CVE-2019-19781) within Citrix ADC (NetScalers). It requests the smb.conf file located in the /vpns/cfg directory by issuing the request /vpn/../vpns/cfg/smb.conf. It then checks if the server is vulnerable by looking for the presense of a "global" variable in smb.conf, which this file should always contain.',
+      'Author'         =>
+        [
+          'Erik Wynter',
+          'altonjx',
```

Comment:
Traditionally, the author's role is provided in comments. ie:
```ruby
[
  'someone', # discovery and PoC
  'someone else', # metasploit
]
```

Reply:
This one's fair.

Reply:
Addressing over in #12816.